cpp/oneapi/dal/backend/memory.hpp

```diff
@@ -16,16 +16,18 @@
 
 #pragma once
 
-#ifdef ONEDAL_DATA_PARALLEL
-#include <CL/sycl.hpp>
-#endif
-
-#include "oneapi/dal/detail/common.hpp"
+#include "oneapi/dal/backend/common.hpp"
 
 namespace oneapi::dal::backend {
 
 #ifdef ONEDAL_DATA_PARALLEL
 
+inline bool is_device_accessible_usm_pointer(const sycl::queue& queue, const void* pointer) {
+    const auto pointer_type = sycl::get_pointer_type(pointer, queue.get_context());
+    return (pointer_type == sycl::usm::alloc::device) || //
```

Comment:
```suggestion
    return (pointer_type == sycl::usm::alloc::device) || 
```

Reply:
This is hint for clang-format to split expression into two lines for better readability. Without `//` clang-format will join the lines.